.github/workflows/publish.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish Packages
+on:
+  pull_request:
+    types:
+      - closed
+    branches:
+      - main
+
+jobs:
+  publish:
+    if: github.event.pull_request.merged == true
```

Comment:
Est-ce que la condition est pas déjà remplie par ce que tu spécifies au dessus dans le on ? Ton job est déjà censé se lancer uniquement quand une PR est merged

Reply:
In the trigger part we check that the PR is `closed` but it might be closed without a merge (for example if it's abandoned)
So here, we also check if the branch was merged